### PR TITLE
chore(Profiles/Notice)

### PR DESCRIPTION
**Pull request by newick**
As discussed, few design tests to improve Users's understanding

### CODE REVIEW

**Comment by newick** [comment]
@gregoirelacoste I added comments to help you understand what i want to do.

**Comment by newick** [comment]
@gregoirelacoste need again some help with 
``` 
{!featured &&  (
          {t('profiles:notice.since', { date: new Date(notice.created) })}
)}
``` 
there's an error but I don't get why

**Comment by gregoirelacoste** [comment]
fixed, translated text wasn't inside a JSX element so it create error

File: src/app/profiles/App/Pages/Profiles/Profile/ProfileIntro.tsx [hunk 1]
```diff
@@ -135,13 +125,7 @@ export const ProfileIntro = ({
...
           avatarSize="large"
           usernameAs={usernameAs}
-        >
-          {contributor && !!contributor.website && (
-            <ExternalLink href={contributor.website}>
-              <LinkIcon /> {contributor.website}
-            </ExternalLink>
-          )}
-        </ContributorLarge>
+        ></ContributorLarge>
```
Comment (gregoirelacoste) [inline src/app/profiles/App/Pages/Profiles/Profile/ProfileIntro.tsx:128]: A simplifier pour :
``` js
<ContributorLarge
          contributor={contributor}
          onSubscribe={subscribe}
          onUnsubscribe={unsubscribe}
          loading={loading}
          avatarSize="large"
          usernameAs={usernameAs}
        />
```

**Review by gregoirelacoste — CHANGES_REQUESTED** [review]

File: src/app/profiles/App/Pages/Profiles/Profile/ProfileIntro.tsx (hunk 1 above)
Comment (newick) [reply src/app/profiles/App/Pages/Profiles/Profile/ProfileIntro.tsx]: that's why webstorm was highlighting it =D

**Comment by JalilArfaoui** [comment]
@newick I don’t understand the 51 commits … do you need help to tidy this up ?

**Comment by newick** [comment]
@JalilArfaoui yes ! Maybe we could speak about it Thursday morning?!

**Comment by JalilArfaoui** [comment]
Totally!

**Comment by gregoirelacoste** [comment]
> @JalilArfaoui yes ! Maybe we could speak about it Thursday morning?!



> @JalilArfaoui yes ! Maybe we could speak about it Thursday morning?!

for me to ? you send an invit ??

**Comment by newick** [comment]
@gregoirelacoste i added you but it's about Webstorm rebase, so up to you if you want to join ;)

**Comment by gregoirelacoste** [comment]
@newick oops wrong PR

**Comment by JalilArfaoui** [comment]
@newick I fixed the history … 8 commits now !

**Review by JalilArfaoui — APPROVED** [review]

**Comment by newick** [comment]
@JalilArfaoui LGTM! thx 🙇

**Review by gregoirelacoste — APPROVED** [review]
lgtm

**Comment by gregoirelacoste** [comment]
@newick do you merge ?

**Comment by newick** [comment]
> @newick do you merge ?

never!

**Comment by newick** [comment]
@gregoirelacoste tu peux gérer le merge ?

**Review by lutangar — APPROVED** [review]

**Pull request merged**